modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.tsx

```diff
@@ -0,0 +1,169 @@
+/* eslint-disable @liferay/empty-line-between-elements */
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import './index.scss';
+
+import {ClayButtonWithIcon} from '@clayui/button';
+
+// import {ClayButtonWithIcon} from '@clayui/button';
+
+import ClayIcon from '@clayui/icon';
+import ClayTable from '@clayui/table';
+
+// import classnames from 'classnames';
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const HEADERS = [
+	{
+		key: 'claim',
+		value: 'Claim Number',
+	},
+	{
+		key: 'property',
+		value: 'Property / Vehicle',
+	},
+	{
+		key: 'date',
+		value: 'Date Filed',
+	},
+	{
+		key: 'fullName',
+		value: 'Name',
+	},
+	{
+		key: 'status',
+		value: 'Status',
+	},
+];
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const activeClaimsData: {[keys: string]: string}[] = [
+	{
+		claim: '816644',
+		date: '11/20/2022',
+		fullName: 'Maria Santos',
+		property: '2019 Ford Edge Se',
+		status: 'Repair',
+	},
+	{
+		claim: '322751',
+		date: '04/30/2022',
+		fullName: 'Maria Santos',
+		property: '2019 Ford Edge Se',
+		status: 'Pending Settlement',
+	},
+	{
+		claim: '322751',
+		date: '04/30/2022',
+		fullName: 'Maria Santos',
+		property: '2019 Ford Edge Se',
+		status: 'Pending Settlement',
+	},
+];
+
+const PolicyActiveClaims = () => {
+	return (
+		<>
+			<div className="bg-neutral policy-active-claims-container rounded">
+				<div className="bg-neutral-0 policy-active-claims-title pt-3 px-5 rounded-top">
+					<h5 className="m-0">Active Claims</h5>
+				</div>
+				<hr className="my-0" />
+				<ClayTable
+					borderedColumns={false}
+					borderless
+					className="tableeee w-100"
+					hover={false}
+				>
+					<ClayTable.Head>
+						<ClayTable.Row>
+							{HEADERS.map((header, index) => (
+								<ClayTable.Cell
+									className="border-bottom py-0 text-paragraph-sm"
+									headingCell
+									key={index}
+								>
+									{header.value}
+								</ClayTable.Cell>
+							))}
+						</ClayTable.Row>
+					</ClayTable.Head>
+					<ClayTable.Body>
+						{activeClaimsData.map((rowContent, rowIndex) => (
+							<>
```

Comment:
Mapping an array without adding a `key` property to the first child will throw an error message in the console, like this: 

![image](https://user-images.githubusercontent.com/22279592/203639827-de498bb1-ad6e-41ea-8912-65978cf9849e.png)

You can use Fragment Component to add a key

```suggestion
							<Fragment key={index}>
```